features/backend/products.feature

```diff
@@ -21,6 +21,10 @@ Feature: Products
             | Black T-Shirt | 19.99 | T-Shirt size                | T-Shirt fabric:Cotton |
             | Mug           | 5.99  |                             |                       |
             | Sticker       | 10.00 |                             |                       |
+          And there are following association types:
+              | name       | code |
+              | Cross sell | PAs1 |
```

Comment:
The convention in a file is to write them explicitly. I think that it will be better to be consistent.